src/rebar_fetch.erl

```diff
@@ -61,7 +61,7 @@ download_source(AppDir, Source, State) ->
     catch
         C:T ->
             ?DEBUG("rebar_fetch exception ~p ~p ~p", [C, T, erlang:get_stacktrace()]),
-            throw(?PRV_ERROR({fetch_fail, Source}))
+            ?PRV_ERROR({fetch_fail, Source})
```

Comment:
> I think the changes were good, but they needed to be followed with changes in rebar_install_deps (the call site) to rethrow the whole tuple instead of just the reason.

Sorry I only caught this reply when I got the email merge notification. I'll have a look at it today.